src/textual/css/tokenizer.py

```diff
@@ -34,9 +34,9 @@ def __init__(
         Args:
             read_from: The location where the CSS was read from.
             code: The code being parsed.
-            start: Line number of the error.
+            start: Line and column number of the error (1-indexed).
             message: A message associated with the error.
-            end: End location of token, or None if not known.
+            end: End location of token (1-indexed), or None if not known.
```

Comment:
It seems like this was 0-indexed before, but now you've converted it to 1 indexed by ensuring we subtract 1 from all the places it was used. What do you think about keeping it zero indexed? Am I missing some context?

Reply:
I don't mind keeping everything 0-indexed and converting only when about to print, but since the errors are created to be user-facing, I thought it made sense to make them 1-indexed from the get-go.

Reply:
As a rule of thumb, the best choice is the one that results in less `-1` and `+1` in your code.

Reply:
I think this doesn't really matter either way, just wanted to be sure we're being consistent